am-lib/src/main/java/uk/gov/hmcts/reform/amlib/AccessManagementService.java

```diff
@@ -122,4 +125,36 @@ public FilterResourceResponse filterResource(String userId, String resourceId, J
 
         return null;
     }
+
+
+    /**
+     * Retrieves a list of {@link RoleBasedAccessRecord } and returns attribute and permissions values.
+     *
+     * @param serviceName  name of service
+     * @param resourceType type of resource
+     * @param resourceName name of a resource
+     * @param roleNames    A set of role names. Currently only one role name is supported but
+     *                     in future implementations we shall support having multiple role names
+     * @return permissionsByTypeAndRole a map of attributes and their corresponding permissions or null
+     */
+    @SuppressWarnings("PMD")
+    //Magic number used temporarily until future implementation supports multiple roles.
```

Comment:
Please delete it, it does not add any value for people reading code. You could have put pull-request comment to share with other reviewers that this is temporary solution.

Reply:
👍